notification/message.go

```diff
@@ -22,3 +27,31 @@ const (
 	MessageTypeAlertBundle
 	MessageTypeAlertStatusBundle
 )
+
+func (s MessageType) Value() (driver.Value, error) {
+	switch s {
+	case 0:
+		return "alert_notification", nil
+	case 1:
+		return "alert_status_update", nil
+	case 2:
+		return "test_notification", nil
+	case 3:
+		return "verification_message", nil
+	case 4:
+		return "alert_notification_bundle", nil
+	case 5:
+		return "alert_status_update_bundle", nil
+	}
+	return nil, fmt.Errorf("could not process unknown type for %s", s)
+}
+
+func (s *MessageType) Scan(value interface{}) error {
+	switch t := value.(type) {
+	case int:
```

Comment:
The type from the DB will be a `string` not an `int`, this will need to have a switch statement like `Value` but inverted.

Here's a suggestion to start:
```go
str, ok := value.(string)
if !ok {
  return fmt.Errorf("could not process unknown type %t", value)
}

switch str {
  case "alert_notification_bundle":
    *s = MessageTypeAlertBundle
  // ...
```